ansible_collections/arista/avd/plugins/test/contains.py

```diff
@@ -60,59 +75,12 @@
 ---
 _value:
   description:
-    - Returns `False` if either the passed value or the test_values are `Undefined` or `none`.
+    - Returns `False` if either the passed value or the test_values are `Undefined` or `None`.
```

Comment:
This is a jinja filter where `none` is the correct casing.

Reply:
[d73ec48](https://github.com/aristanetworks/avd/pull/4131/commits/d73ec488af3c37b61ca939811bc129671e44242b)